inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -75,6 +77,58 @@ def from_text_file(
         depths = np.loadtxt(text_file_path)
         return cls(depths, x_range, y_range)
 
+    @classmethod
+    def from_random_depths(
+        cls,
+        n: int,
+        x_range,
+        y_range,
+        max_depth: float = 10,
+        initial_roughness: float = 1,
+        roughness_factor: float = 0.5,
+        percentile_above_water: float = 20,
+    ):
+        """Creates a `Bathymetry` object with random depths.
+
+        A grid of random depths of size (2^n + 1) x (2^n + 1) is created, where
+        n is an integer.
+
+        The depths are randomly generated as follows:
+        1. A maximum depth value `max_depth` is used to specify the range of
```

Comment:
With the description below this seems to be `max_amplitude` or something, since depth seems more related to below sea level. Wouldn't it make sense to have two different parameters for both, so that we can have plan beaches with deep waters for example.